### PR TITLE
Addids

### PR DESCRIPTION
**Pull request by WillyStorm**
Add moving content, alt text for CSS background images, resize, and accessibility.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,16 +13,16 @@
...
                     </ul>
                 </nav>
             </div>
         </header>
     </div>  
-        <section class="padd">
+        <section class="padd" id="Hello">
             <div class="gridbox">
                 <div class="grid1">
                 <h1>Charlie<br>Giorgania</h1>
```
Comment (keroloussamy) [inline index.html:28]: - [ ] Kindly here your headline overlaps the image, kindly fix it to match the requirements with the `Resize Text` point.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly you have to add a report to your pull request, as the requirements say.

image

- [ ] Kindly update your page's title with something like `Mark's portifolio`, it will be better.
https://github.com/WillyStorm/PortFolioMobile/blob/3ceee091ad3ee837777c5b7a10a7e5cd4f6f4188/index.html#L8

- [ ] Kindly change your alt attribute, to a descriptive one like `twiter account` instead of `social`, kindly change them all.
https://github.com/WillyStorm/PortFolioMobile/blob/3ceee091ad3ee837777c5b7a10a7e5cd4f6f4188/index.html#L212-L219



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by WillyStorm** [comment]
List all points checked, i.e:

    Page titles 
                - changes made: 
https://github.com/WillyStorm/PortFolioMobile/blob/70bf11fb9d4eef210f91aad2fd20da42aad0b825/index.html#L8

                - no issues found

    Image text alternatives** (alt text) 
               - changes made: 
https://github.com/WillyStorm/PortFolioMobile/blob/70bf11fb9d4eef210f91aad2fd20da42aad0b825/index.html#L213-L218
               -  no issues found
   
 Text heading
              -changes made:
https://github.com/WillyStorm/PortFolioMobile/blob/70bf11fb9d4eef210f91aad2fd20da42aad0b825/style.css#L68
https://github.com/WillyStorm/PortFolioMobile/blob/70bf11fb9d4eef210f91aad2fd20da42aad0b825/style.css#L429-L430
            
                 - no issues found
             
    Color contrast - no issues found
    Resize - no issues found
    Interaction - no issues found
    Moving content - no issues found
    Multimedia - no issues found
    The basic structure of the page - no issues found

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**